internal/net/grpc/pool/pool.go

```diff
@@ -356,28 +355,32 @@ func (p *pool) Do(f func(conn *ClientConn) error) error {
 }
 
 func (p *pool) Get() (*ClientConn, bool) {
-	return p.get(p.Len())
-}
-
-func (p *pool) get(retry uint64) (*ClientConn, bool) {
-	if retry <= 0 || retry > math.MaxUint64-p.Len() || p.Len() <= 0 {
```

Comment:
Do anyone know what is the checking `retry > math.MaxUint64-p.Len()` means here? I have omitted it on my refactoring.

Reply:
@kpango do you know what is the meaning of this logic? 
I wanted to refactor it but I can't understand what it means, so I couldn't complete this refactoring for now.